website/docs/r/sns_topic.html.markdown

```diff
@@ -54,40 +54,52 @@ resource "aws_sns_topic" "user_updates" {
+The `<endpoint>_success_feedback_role_arn` and `<endpoint>_failure_feedback_role_arn` arguments are used to give Amazon SNS write access to use CloudWatch Logs on your behalf. The `<endpoint>_success_feedback_sample_rate` argument is for specifying the sample rate percentage (0-100) of successfully delivered messages. After you configure the `<endpoint>_failure_feedback_role_arn` argument, then all failed message deliveries generate CloudWatch Logs.
 
 ## Argument Reference
 
 The following arguments are supported:
 
-* `name` - (Optional) The friendly name for the SNS topic. By default generated by Terraform.
-* `name_prefix` - (Optional) The friendly name for the SNS topic. Conflicts with `name`.
-* `display_name` - (Optional) The display name for the SNS topic
-* `policy` - (Optional) The fully-formed AWS policy as JSON. For more information about building AWS IAM policy documents with Terraform, see the [AWS IAM Policy Document Guide](https://learn.hashicorp.com/terraform/aws/iam-policy).
-* `delivery_policy` - (Optional) The SNS delivery policy. More on [AWS documentation](https://docs.aws.amazon.com/sns/latest/dg/DeliveryPolicies.html)
-* `application_success_feedback_role_arn` - (Optional) The IAM role permitted to receive success feedback for this topic
-* `application_success_feedback_sample_rate` - (Optional) Percentage of success to sample
-* `application_failure_feedback_role_arn` - (Optional) IAM role for failure feedback
-* `http_success_feedback_role_arn` - (Optional) The IAM role permitted to receive success feedback for this topic
-* `http_success_feedback_sample_rate` - (Optional) Percentage of success to sample
-* `http_failure_feedback_role_arn` - (Optional) IAM role for failure feedback
-* `kms_master_key_id` - (Optional) The ID of an AWS-managed customer master key (CMK) for Amazon SNS or a custom CMK. For more information, see [Key Terms](https://docs.aws.amazon.com/sns/latest/dg/sns-server-side-encryption.html#sse-key-terms)
-* `lambda_success_feedback_role_arn` - (Optional) The IAM role permitted to receive success feedback for this topic
-* `lambda_success_feedback_sample_rate` - (Optional) Percentage of success to sample
-* `lambda_failure_feedback_role_arn` - (Optional) IAM role for failure feedback
-* `sqs_success_feedback_role_arn` - (Optional) The IAM role permitted to receive success feedback for this topic
-* `sqs_success_feedback_sample_rate` - (Optional) Percentage of success to sample
-* `sqs_failure_feedback_role_arn` - (Optional) IAM role for failure feedback
-* `tags` - (Optional) Key-value map of resource tags
+- `name` - (Optional) The friendly name for the SNS topic. By default generated by Terraform.
```

Comment:
Could you please revert to `*` for list indicator so as to minimize the diffs? Thanks.

Reply:
https://github.com/hashicorp/terraform-provider-aws/pull/15828/commits/0d044779792e017832f52e87e17836cf65e44a42